src/components/metadata/metadata-row.js

```diff
@@ -19,7 +21,7 @@ const MetaDataRow = ({
   children,
 }) => {
   const showList = Array.isArray(value);
-
+  const showObject = typeof value === 'object' && !showList;
```

Comment:
I am not sure of the need for the `!showList` check as showList would not be positive if `value` is an object. 
I would, however, add an additional check to ensure that the object is not an object ( i.e `const showObject = typeof value === 'object' && Object.keys(value).length > 0`) before displaying it.

Reply:
@studioswong  -- it's weird but if you do type of array in javascript it's an object, hence showList is also true for objects. however it's not true for object with key value pairs -- hence I think this works the same way as your suggested way of implementation - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
aargh I see - good point! yea in that case let's keep this